Decred Wallet/Extensions/UIColor.swift

```diff
@@ -26,6 +26,8 @@ extension UIColor {
         static let thinGray = UIColor.init(hex: "#a4abb1")
         static let darkGray = UIColor.init(hex: "#4e5f70")
         static let transparentThinGray = UIColor.init(hex: "#a4abb1", alpha: 0.3)
+        static let textDark = UIColor.init(hex: "#596d81")
```

Comment:
Other colors here are more descriptive: `offWhite`, `lightGray`, `thinGray`. `textDark` doesn't tell us what color this is: it could be dark yellow or dark blue. Also, if there's a color already defined that's close to this, use that instead. We shouldn't be bombarding our app with so many color variants.

Reply:
removed in 2672a92